project/src/components/not-found-screen/not-found-screen.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import Logo from '../logo/logo';
+
+function NotFoundScreen() {
+  return (
+    <div className="user-page">
+      <header className="page-header user-page__head">
+        <Logo />
+      </header>
+
+      <section className="not-found__screen">
+        <h1>404. Page not found</h1>
+        <Link to="/">Вернуться на главную</Link>
+      </section>
+
+      <footer className="page-footer">
+        <Logo isFooterLogo={true} />
```

Comment:
```suggestion
        <Logo isFooterLogo />
```
Если пропс булевый, и должен быть тру, то можно передать просто название